doc/dev/installation.rst

```diff
@@ -2,7 +2,7 @@ Installation
 ============
 
 
-Catalyst is officially supported on Linux (x86_64) and macOS (aarch64, x86_64) platforms, and
+Catalyst is officially supported on Linux and macOS, on both x86_64 and aarch64 platforms, and
```

Comment:
This might be minor, but I understand platform as both os + arch, or maybe either of them but not when they are both listed. 

It might also make sense to write `aarch64/arm64` for mac people who might have heard of arm but not of aarch :)

Reply:
Done